gateway-service/src/test/resources/gateway-messages.yml

```diff
@@ -328,10 +328,13 @@ messages:
       number: ZWEAG182
       type: ERROR
       text: >
-        SSL Misconfiguration, z/OSMF is not accessible. Please verify the following: \n
-          - CN (Common Name) and z/OSMF hostname match\n
-          - The certificate is expired\n
-          - TLS version matches\n
-        Enable debugging to see further details in stack trace
-      reason: "z/OSMF connection has an incorrect configuration"
+        SSL Misconfiguration, z/OSMF is not accessible.
+        Message: %s
+        Please verify the following:
+          - CN (Common Name) and z/OSMF hostname match.
+          - The certificate is valid
+          - TLS version matches
+          - z/OSMF server certificate is trusted in Zowe's truststore
+        Enable debugging to see further details in stack trace.
```

Comment:
Maybe we could suggest to enable SSL debugging specifically

Reply:
I added it to the details of the action, @janan07 is is ok for you?

Reply:
Looks good.